.changeset/real-vans-provide.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/cli': patch
+---
+
+Remove `apiUrl` from the output of the create-github-app because apiUrl already exist in the GitHub integration config.
```

Comment:
No need for a changeset if we merge this before the next release, it just modifies a change that is already a part of the pending changesets